### DISCUSSION TITLE
Problem parsing longer BC messages

### DISCUSSION THREAD

**Discussion by vadito** [john30/ebusd#952] [Q&A]
Hi all,
I got my v5 adapter this week and started working on getting my Ariston furnace up and running. I'm using a csv found on github for a similar(but not the same) Ariston device as a base.
For most parts my modified CSV works fine but I'm struggling with parsing some BC messages which I can see in the ebusd log but are claimed to be unknown.
Maybe someone can give me an idea what am I missing. I've also noticed a pattern, shorted broadcasts are parsed successfully and work fine(verified with 5B payload) but longer ones like the 8B below have the same problem.

Message in ebus log:
`[update notice] received unknown BC cmd: 37fe200e086271b40064002c01`

Definition in CSV:
`r,energymgr,z1_day_temp_bcast,Z1 Day temperature,,fe,200e,6271,,s,SIN,10,°C`

**Comment by VoyteckPL** [comment]
r,boiler,day_temp,Day temperature,37,fe,200e,6271,day_temp,s,SIN,,°C,,,,,,,,,,,,,,,,,,,,,,

This line works for me, can you test?
